index.html

```diff
@@ -1470,15 +1556,15 @@ <h2>Expressing Verifiable Credentials in WebIDL</h2>
 </pre>
 
             <p>This <a>verifiable credential expression</a> could then be
-            used as follows in JavaScript:</p>
+            used as follows in JavaScript: STILL TO DO</p>
 
 <pre class="example" title="Using a simple claim">
   var claim = new Claim1();
   claim.id = "http://example.gov/credentials/3732";
   claim.ageOver = 21;
 
   var credential = new Credential();
-  credential.type = ["Credential", "ProofOfAgeCredential"];
+  credential.type = ["verifiableCredential", "ProofOfAgeCredential"];
```

Comment:
Types should start with a capital letter.

Reply:
This was a typo. "verifiableCredentail" should have been deleted entirely.